lib/ASTGen/Sources/ASTGen/Macros.swift

```diff
@@ -430,6 +431,21 @@ func expandAttachedMacro(
         $0.trimmedDescription
       }.joined(separator: "\n\n")
 
+    case (let attachedMacro as PeerMacro.Type, .Peer):
+      let peers = try attachedMacro.expansion(
+        of: sourceManager.detach(
+          customAttrNode,
+          foldingWith: OperatorTable.standardOperators
+        ),
+        providingPeersOf: sourceManager.detach(declarationNode),
```

Comment:
Do we need to fold the declaration node?

Reply:
Ah, we probably do. Should we also fold the declaration node for accessor macros?

Reply:
Yes, I think we should